src/simplewallet/simplewallet.cpp

```diff
@@ -1580,7 +1583,8 @@ void simple_wallet::on_skip_transaction(uint64_t height, const cryptonote::trans
 //----------------------------------------------------------------------------------------------------
 bool simple_wallet::refresh_main(uint64_t start_height, bool reset)
 {
-  if (!try_connect_to_daemon())
+  uint32_t version;
+  if (!try_connect_to_daemon(version))
```

Comment:
Could have been a pointer, defaulting to NULL, to avoid changing all these call sites for non real reason